CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 ### UI Improvements
 1. [#1796](https://github.com/influxdata/chronograf/pull/1796): Add spinner to indicate data is being written
 1. [#1800](https://github.com/influxdata/chronograf/pull/1796): Embiggen text area for line protocol manual entry in Data Explorer's Write Data overlay
+1. [#1805](https://github.com/influxdata/chronograf/pull/1805): Bar graphs no longer overlap with each other, and bonus, series names are hashed so that graph colors should stay the same for the same series across charts
```

Comment:
1. 😺 on `bonus`. but unfortunately, can you separate these into two separate bullets under `UI Improvements` since they're actually unrelated? they can both have the same PR number and link.
1. and for both, could you follow the convention of present tense Verb + change content?